style.css

```diff
@@ -0,0 +1,376 @@
+}
+
+a {
+  text-decoration: none;
+  color: rgba(var(--clr-dark));
+  display: block;
+}
+
+h3,
+h2 {
+  font-weight: 600;
+}
+
+h2 {
+  font-size: 1.8rem;
+}
+
+.container {
+  height: 100vh;
+}
+
+.grey-sub-text {
+  color: rgba(var(--clr-grey));
+}
+
+/* ------------------- */
+/* left styling        */
+/* ------------------- */
+/* nav */
+
+.nav-list > * {
+  display: grid;
+  grid-template-columns: auto 1fr;
+  padding-block: 10px;
+}
+
+li > a:hover,
+li > a:focus {
+  color: rgba(var(--clr-accent));
+}
+
+.nav-list a {
+  padding-left: 20px;
+}
+/* profile shortcut */
+.profile-shortcut {
+  margin-bottom: 20px;
+}
+
+/* ------------------- */
+/* main styling        */
+/* ------------------- */
+
+.main-section {
+  border-inline: rgba(var(--clr-grey-light)) solid 1px;
+  overflow-y: scroll;
+}
+
+.home-title {
+  padding: 10px;
+  border-bottom: rgba(var(--clr-grey-light)) solid 1px;
+}
+
+/* status section */
+
+.status-bar {
+  border: none;
+  padding-left: 10px;
+  margin-block: 10px;
+}
+
+.status-actions {
+  margin-bottom: 20px;
+}
+
+.status-container {
+  border-bottom: 15px rgba(var(--clr-grey-light), 0.2) solid;
+}
+
+/* posts section */
+
+main .icon-md {
+  margin-top: 10px;
+  margin-left: 10px;
+}
+
+.post {
+  border-block: rgba(var(--clr-grey-light)) 1px solid;
+}
+
+.post-img {
+  width: 90%;
+  border-radius: 15px;
+}
+/* ------------------- */
+/* right styling       */
+/* ------------------- */
+
+.right-section-wrapper {
+  margin-left: 1.5rem;
+}
+.right-section-wrapper > * {
+  padding: 20px;
+}
+
+.search-bar {
+  width: 100%;
+  border: none;
+  background-color: rgba(var(--clr-grey-light), 0.2);
+  border-radius: 30px;
+  padding: 10px;
+  margin-block: 10px;
+}
+
+.stories {
+  background-color: rgba(var(--clr-grey-light), 0.2);
+  border-radius: 10px;
+}
+
+.story {
+  border-bottom: rgba(var(--clr-grey)) solid 1px;
+  padding-bottom: 10px;
+  margin-bottom: 10px;
+}
+
+.stories div:last-child {
+  border-bottom: none;
+}
+
+.story-img {
+  width: 100px;
+  border-radius: 10px;
+}
+
+/* responsiveness */
+
+.right-section {
+  display: none;
+}
+
+.tweet-btn,
+.profile-shortcut,
+.nav-list,
+.nav-list a,
+.twitter {
+  display: none;
+}
+
+@media only screen and (min-width: 500px) {
+  .twitter,
+  .nav-list {
+    display: block;
+  }
+
+  .container {
+    display: grid;
+    grid-template-columns: auto 1fr;
+  }
+}
+
+@media only screen and (min-width: 700px) {
+  .twitter,
+  .nav-list,
+  .nav-list a,
+  .tweet-btn {
+    display: block;
+  }
+
+  .container {
+    display: grid;
+    grid-template-columns: 0.3fr 1fr;
+  }
+}
+
+@media only screen and (min-width: 930px) {
+  .profile-shortcut {
+    display: grid;
+    grid-template-columns: auto 1fr auto;
+  }
+}
+
+@media only screen and (min-width: 1000px) {
+  .twitter,
+  .nav-list,
+  .nav-list a,
+  .right-section {
+    display: block;
+  }
+
+  .container {
+    display: grid;
+    grid-template-columns: 0.4fr 1fr 0.5fr;
+  }
+}
```

Comment:
Did you built the page considering mobile first, or is it just easier to visualize your media queries starting from smaller screens?